frontend/src/posapp/layouts/DefaultLayout.vue

```
@@ -517,6 +519,20 @@ const handleToggleOffline = () => {
 	}
 };
 
+const handleRetryStatus = async () => {
+	if (getIsManualOffline()) {
+		toastStore.show({
+			title: __("Manual offline mode is enabled"),
```

ESLint

[error] '__' is not defined.

(no-undef)